live-server.js

```diff
@@ -59,6 +60,13 @@ for (var i = process.argv.length - 1; i >= 2; --i) {
 		opts.logLevel = 0;
 		process.argv.splice(i, 1);
 	}
+	else if (arg.indexOf("--mount=") > -1) {
+		// e.g. "--mount=/components:./node_modules" will be ['/components', '<process.cwd()>/node_modules']
+		var mountRule = arg.substring(8).split(":");
+		mountRule[1] = path.resolve(process.cwd(), mountRule[1]); 
```

Comment:
Hehe, nice you found this, I was thinking about the right way for quite a time. I decided to go with this for the following reason. Imagine your project structure is
```
* my_proj
   * node_modules
   * source
```
And I want the live-server to serve the files in *source*. My command line is located in the root folder of the project. In this case I think it is more intuitive to address the directories relatively to my current position and not to the future position.
```
my_proj$ live-server --mount=/components:node_modules source
```
vs
```
my_proj$ live-server --mount=/components:../node_modules source
```
In addition I can use the path completion of my shell to resolve the paths, because they are relative to my position.